pkgs/by-name/ye/yed/package.nix

```diff
@@ -13,7 +13,8 @@ stdenv.mkDerivation rec {
   version = "3.25";
 
   src = fetchzip {
-    url = "https://www.yworks.com/resources/yed/demo/${pname}-${version}.zip";
+    # to update: archive https://www.yworks.com/resources/yed/demo/${pname}-${version}.zip
+    url = "https://web.archive.org/web/20250212125159/https://www.yworks.com/resources/yed/demo/${pname}-${version}.zip";
```

Comment:
```suggestion
    # to update: archive https://www.yworks.com/resources/yed/demo/yEd-${version}.zip
    url = "https://web.archive.org/web/20250212125159/https://www.yworks.com/resources/yed/demo/yEd-${version}.zip";
```

Reply:
thanks! i squashed to include this change now.